devices/philips.js

```diff
@@ -2019,15 +2019,15 @@ module.exports = [
         vendor: 'Philips',
         description: 'Hue motion outdoor sensor',
         fromZigbee: [fz.battery, fz.occupancy, fz.temperature, fz.illuminance, fz.occupancy_timeout,
-            fz.hue_motion_sensitivity, fz.hue_motion_led_indication],
+            fz.hue_motion_sensitivity_2, fz.hue_motion_led_indication],
         exposes: [e.temperature(), e.occupancy(), e.battery(), e.illuminance_lux(), e.illuminance(),
-            exposes.enum('motion_sensitivity', ea.ALL, ['low', 'medium', 'high']),
+            exposes.enum('motion_sensitivity', ea.ALL, ['very low', 'low', 'medium', 'high', 'very high']),
             exposes.binary('led_indication', ea.ALL, true, false).withDescription('Blink green LED on motion detection'),
             exposes.numeric('occupancy_timeout', ea.ALL).withUnit('second').withValueMin(0).withValueMax(65535)],
-        toZigbee: [tz.occupancy_timeout, tz.hue_motion_sensitivity, tz.hue_motion_led_indication],
+        toZigbee: [tz.occupancy_timeout, tz.hue_motion_sensitivity_2, tz.hue_motion_led_indication],
         endpoint: (device) => {
             return {
-                'default': 2, // default
+                'default': 4, // default
```

Comment:
Can you confirm this does not break setting the `occupancy_timeout` and `led_indication`?

Reply:
Adding endpoint 4 was an error. I thought it was value by default. So yes it broke the device.